### ISSUE TITLE
DDRIT: Auto-inserted text

### ISSUE THREAD

**Issue by TanayParikh** [dotnet/aspnetcore#26976]
Big Rock: https://github.com/dotnet/aspnetcore/issues/23646

**Comment by TanayParikh** [comment]
Is this:

```
<strong>|
```

->

```
<strong>|<strong>
```
?

**Comment by NTaylorMullen** [comment]
Yes, that tests HTML. then `@*` -> `@* | *@` tests Razor then `///` -> `/// <summary> ... etc.` tests C#

**Comment by TanayParikh** [comment]
Created: https://devdiv.visualstudio.com/DevDiv/_git/VS/pullrequest/285648?_a=files

**Comment by TanayParikh** [comment]
Blocked on https://github.com/dotnet/aspnetcore-tooling/pull/2697

**Comment by TanayParikh** [comment]
Blocked on: https://github.com/dotnet/roslyn/pull/48468

**Comment by TanayParikh** [comment]
Just merged, should be unblocked after Roslyn inserts.

**Comment by TanayParikh** [comment]
Blocked on DDRIT issues.

**Comment by TanayParikh** [comment]
This issue remains blocked on [diagnostics not being await-able in the DDRIT environment](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1253831). The plan is to leverage diagnostics being available as a way for us to know if the language server(s) have initialized. Only thereafter do we actually perform this test.

**Comment by TanayParikh** [comment]
This issue remains blocked on [diagnostics not being await-able in the DDRIT environment](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1253831). Due to external re-prioritization, the blocking bug will not be addressed till 16.10.

**Comment by TanayParikh** [comment]
Still blocked on [diagnostics not being await-able in the DDRIT environment](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1253831).
